File: app/inputuser.py
class InputUser:
    """Get input user."""

    def __init__(self, raw_input):
        """Create an input user object.

        Args:
            raw_input (str): A raw input from user

        Attribute:
            raw_input (str): Contain the raw input from user
        """
        self._raw_input = raw_input

    @property
    def raw_input(self):
        return self._raw_input

File: app/static/scripts/modules/InputUser.js
export class InputUser {
  /* Create an InputUser object and parse it into JSON.
    */

  constructor (inputUser) {
    /* Turn the input user into a javascript object.

    Args:

        inputUser (object): Raw user input into JS object.

    */

    this._inputUser = {
      sentence: inputUser
    }
  }

  getInputUser () {
    return this._inputUser
  }

  setInputUser (inputUser) {
    this._inputUser.sentence = inputUser
  }

  IntoJSON () {
    /* Parse the input user into a JSON string.

    Parsing occur in place. */

    this._inputUser = JSON.stringify(this._inputUser)
  }
}
